### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by mj-e-boc**
-Add nav section for front page, 
-Add main section
-Add styles for front page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,145 @@
...
+
+### Install
+
+Install this project
+
+### Usage
+
+To run the project:
+
+click on go Live on the bottom of VsCode, and view it on the screen.
```
Comment (muneebulrehman) [inline README.md]: Kindly remove these sections from your README file to keep it clean since these sections are empty and seem irrelevant to this project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,145 @@
...
+
+- **[You_open_for_new_ideas]**
+
+  - [The_more_the_better]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (muneebulrehman) [inline README.md]: The link to the MIT license is broken. Kindly add a valid license link.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- mobile header -->
+    <header>
+      <nav>
+        <h1>My Logo</h1>
```
Comment (muneebulrehman) [inline index.html]: 1. I would recommend moving the logo out of the `nav` component since the logo is not a part of the navbar. Kindly do so, before submitting for a re-review.
2. Let's wrap the logo in an `a` tag instead of the `h1` element, so when visitors click the logo, they are redirected to the top/home page of your application.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        <div class="bars">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+      </nav>
+    </header>
+    <!-- mobile main section -->
+    <main>
```
Comment (muneebulrehman) [inline index.html:41]: Let's remove this `<main>` tag as it serves no purpose here. The same work will be done by the section underlying it.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      <section class="frontpage">
+        <h2>Hey There. I’m Ronald.</h2>
```
Comment (muneebulrehman) [inline index.html]: Let's make this heading an `h1` since introducing yourself is the most important part of your portfolio and `h1` is reserved for the same.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        <h3>I'm a Software Developer</h3>
```
Comment (muneebulrehman) [inline index.html]: Let's make this `<h3>` a `<p>` tag. This way will be semantically correct and the screen readers and other technologies will thank you.
At the same time, kindly use the `font-family: 'Roboto'` as given in the Figma template of your choosing.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
```
Comment (muneebulrehman) [inline index.html:49]: As per the Figma template of your choosing, this `p` tag has `font-family: 'Poppins';` instead of `Creta Round` that you are using. Kindly update accordingly.
- I would also recommend using a `line-height: 24px` to increase the gap between the lines.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+            <img src="./images/evangelist.svg" alt="evangelisticon" />
+          </li>
+          <li>
+            <img src="./images/twitter.svg" alt="twittericon" />
+          </li>
+          <li>
+            <img src="./images/myspace.svg" alt="myspaceicon" />
+          </li>
+        </ul>
+      </footer>
```
Comment (muneebulrehman) [inline index.html]: Let's not use the `<footer>` tag here as it is reserved for the footer of an application and I am sure you can also tell that this is not going to be the footer of this application. You can remove the `<footer>` and incarporate this list of icons inside this `<section class="frontpage">` tag

File: index.html [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+          </li>
+          <li>
+            <img src="./images/evangelist.svg" alt="evangelisticon" />
+          </li>
+          <li>
+            <img src="./images/twitter.svg" alt="twittericon" />
+          </li>
+          <li>
+            <img src="./images/myspace.svg" alt="myspaceicon" />
+          </li>
```
Comment (muneebulrehman) [inline index.html]: Each of the icons need to be inside an `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @mj-e-boc , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job with trying to replicate the exact Figma design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 10]
```diff
@@ -1,2 +1,136 @@
...
+- **[You_open_for_new_ideas]**
+
+  - [The_more_the_better]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
```
Comment (Owusu-Desmond) [inline README.md]: - [ ] Please make sure you add a `LICENSE` file to your project and direct the license link in your readme file to the license file as requested by the previous code reviewer. Currently, your license file is not broken but it is not directed to your own created license :blush:

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+header a {
+  margin-right: auto;
+  margin-left: 5%;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  font-family: "Crete Round", serif;
+  font-family: "Poppins", sans-serif;
```
Comment (Owusu-Desmond) [inline styles.css]: - [ OPTIONAL ] I think that you should use the `font-family: "Roboto", sans-serif;` which is the preferred one in the Figma design, it would be better if you remove `font-family: "Crete Round", serif;` because you can not have two repeated CSS properties :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @mj-e-boc  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 12]
```diff
@@ -1,2 +1,136 @@
...
+- **[You_open_for_new_ideas]**
+
+  - [The_more_the_better]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```
Comment (Ibizugbe) [inline README.md]: I can see you created a license file as requested by the previous reviewer👍. But currently, you still haven't updated the license link🥲. Kindly update the license link with the link to the created license file in your repo😉. When this is implemented, it should lead to your license file😉.


```suggestion
This project is [MIT](./LICENSE) licensed.
```

Try the suggestion above if it works, but I think that should work😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @mj-e-boc 👋👋,

tryagain
 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅

- [x] Linters are passing👍.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ofarouq310 — APPROVED** [review]
Hi @mj-e-boc ,✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Descriptive commit messages.
✅ No linters errors.
✅ Well formatted README file.

## Optional suggestions

- Please make sure your commit messages are always capitalized and in the imperative tense.


- Your branch should have a name of a feature or action you are implementing, in your case it could be "mobile-setup" or "header-and-deadline" or whatever suits you. Kindly fix it and rename your branch. Kindly refer to the next image for the steps of renaming a branch.
image


- Your Pull request title should be short, and concise, and addresses what you are implementing, in your case, it could be, `Add header and headline sections` or something similar, simpler. Please refer to the next picture to help you how to edit the title.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
